disabled.yml

```diff
@@ -0,0 +1,2 @@
+AllCops:

```

Comment:
Done.  I was working with Mike (mentor) to fix error before I discovered it was just the one page.